targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_MCU_K24F/TARGET_MCU_K24F1M/device/TOOLCHAIN_GCC_ARM/MK24FN1M0xxx12.ld

```diff
@@ -56,16 +56,23 @@ __ram_vector_table__ = 1;
 __stack_size__ = 0x8000;
 __heap_size__ = 0x10000;
 
+#if !defined(MBED_APP_START)
```

Comment:
This is adding bootloader to mk24, should be separate commit for this change.

Reply:
Just a separate commit in this change set or new pull request?

Reply:
I would send a new PR, then this would need to be rebased on top of it  (once the first one gets integrated). In this case, can be as a commit, added here.

Reply:
Thank you! And done.